Gruntfile.js

```diff
@@ -115,11 +143,26 @@ module.exports = function(grunt) {
                 ],
                 module: {
                     loaders: [
-                        {
-
-                            test: /\.less$/,
-                            loader: 'style-loader!css-loader!less-loader'
-                        }
+						{
```

Comment:
Setup indentation and other style guide rules per https://github.com/jwplayer/jwplayer#style-guide